src/client/components/card/CardModal.tsx

```diff
@@ -133,6 +129,18 @@ const CardModal: React.FC<CardModalProps> = ({
     setImageUsed(getCardBackImage(card));
   }
 
+  const updateField = useCallback(
+    (field: keyof Card, value: any) => {
+      //Handle edge case of the back image being removed while looking at the back
+      if (field === 'imgBackUrl' && value.trim() === '' && !isFrontImage) {
```

Comment:
Without this you could look at the custom back image, remove it, then see the MTG card back with no way to see the front face. That applies to a card which normally has only a front face, but the custom back image allowed the transform.